src/WebJobs.Script/Workers/Rpc/Configuration/WorkerDescriptionProfileConverter.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers
+{
+    public class WorkerDescriptionProfileConverter : JsonConverter
+    {
+        private ILogger _logger;
```

Comment:
Check if these can be dependency injected?

Reply:
These 3 should already be dependency injection in the project so they should be available for you to use